bitrise.yml

```diff
@@ -168,21 +168,23 @@ workflows:
         - xctestrun: $BITRISE_TEST_BUNDLE_PATH/Fennec_UnitTest_iphonesimulator18.2-arm64.xctestrun
         - maximum_test_repetitions: 2
     - xcode-test-shard-calculation@0:
+        run_if: '{{getenv "BITRISE_XCTESTRUN_SMOKE_TEST_FILE_PATH" | ne ""}}'
         inputs:
         - product_path: $BITRISE_XCTESTRUN_SMOKE_TEST_FILE_PATH
         - shard_count: 5
         - destination: $BITRISE_DESTINATION
         outputs:
         - BITRISE_TEST_SHARDS_PATH: BITRISE_TEST_SHARDS_PATH_FIREFOX 
+        is_always_run: true
```

Comment:
We need `is_always_run` when the unit tests fails but we want to shards to be processed.

Reply:
I don't think we need this or the run_if in 171, actually, we are saying run_if and also always run... as long as we control when the deploy step runs, we are fine. This part was working correctly if I'm not mistaken, the confusing part is with deploy step.